frontend/src/plugins/remote/socket/index.ts

```diff
@@ -43,24 +41,24 @@ const socketUrl = computed(() => {
 });
 
 const { data, send } = useWebSocket(socketUrl, {
-  heartbeat: { message: formatSocketMessage('KeepAlive') },
+  heartbeat: false,
   autoReconnect: { retries: () => true },
   immediate: true,
   autoClose: false
 });
 
 class RemotePluginSocket {
-  public message = computed<WebSocketMessage | undefined>(() => {
-    const m = destr(data) as WebSocketMessage;
-
-    return typeof m.MessageType === 'string' ? m : undefined;
-  });
-  public messageType = computed<WebSocketMessage['MessageType'] | undefined>(
-    () => this.message.value?.MessageType
-  );
-  public messageData = computed<WebSocketMessage['Data'] | undefined>(
-    () => this.message.value?.Data
-  );
+  public get message(): WebSocketMessage | null {
+    return destr(data.value) as WebSocketMessage | null;
+  }
+  public get messageType(): WebSocketMessage['MessageType'] | undefined {
+    return this.message && this.message.MessageType
+      ? this.message.MessageType
+      : undefined;
+  }
+  public get messageData(): WebSocketMessage['Data'] | undefined {
+    return this.message && this.message.Data ? this.message.Data : undefined;
+  }
```

Comment:
Does it make sense to have these 3 values separate? In the instances they're used, there is always a watcher on `message` and then the values used are the Data and MessageType, but those can already easily be checked in a single line using `message?.MessageType` and `message?.Data`.

I think only `message` as a computed value needs to be kept around, the other 2 don't provide much utility IMO